### PR TITLE
Fixed input/output shape initialization

### PR DESCRIPTION
**Pull request by ilyachur**

### CODE REVIEW

File: docs/template_extension/cpu_kernel.cpp [hunk 1]
```diff
@@ -23,6 +23,8 @@ OpImplementation::OpImplementation(const std::shared_ptr<ngraph::Node> &node) {
         if (castedNode->get_input_element_type(0) != ngraph::element::f32 || castedNode->get_output_element_type(0) != ngraph::element::f32)
             THROW_IE_EXCEPTION << "Operation supports only FP32 tensors.";
         add = castedNode->getAddAttr();
+        inShape = castedNode->get_input_shape(0);
```
Comment (dkurt) [inline docs/template_extension/cpu_kernel.cpp:26]: What if there are more that one input?

Comment (ilyachur) [reply docs/template_extension/cpu_kernel.cpp:26]: We have a check in this line: https://github.com/openvinotoolkit/openvino/pull/1695/files#diff-7b34831df09cb077dde39d1b3c043a46R17

Comment (ilyachur) [reply docs/template_extension/cpu_kernel.cpp:26]: @dkurt Is this PR work for your case?

Comment (dkurt) [reply docs/template_extension/cpu_kernel.cpp:26]:
```
terminate called after throwing an instance of 'InferenceEngine::details::InferenceEngineException'
  what():  Cannot fill descriptor. Size of dimensions and order vector don't match.
Aborted (core dumped)
```
for now. Will try to check the total diff comparing to master

**Comment by ilya-lavrenov** [comment]
@ilyachur we have `extensions_tests` which is used in tests. Can we merge these 2 libs into 1 and re-use `template_extension` in tests? In this case template library will be checked for correctness

File: docs/template_extension/cpu_kernel.cpp (hunk 1 above)
Comment (ilyachur) [reply docs/template_extension/cpu_kernel.cpp:26]: Ok, It happens because we hardcoded the order. I will modify the code.

**Comment by ilyachur** [comment]
> @ilyachur we have `extensions_tests` which is used in tests. Can we merge these 2 libs into 1 and re-use `template_extension` in tests? In this case template library will be checked for correctness

I'll try to do that

File: docs/template_extension/cpu_kernel.cpp (hunk 1 above)
Comment (dkurt) [reply docs/template_extension/cpu_kernel.cpp:26]: Actually, this appears only after network's reshape. Origin shapes inference works fine, thanks!

**Pull request merged**